src/@types/Core.d.ts

```diff
@@ -81,6 +92,9 @@ declare global {
     requests: Array<CoreRequest>;
     resquestsMetaDataById: CoreRequestMetaDataById;
     selectedRequest?: CoreRequest;
+    settings: {
```

Comment:
We have a new core state called settings, which is an object and in this first version, it contains only the URLs arrays. It is already an object so we can grow easily without the need to change anything else